set_1_basics/challenge_1_convert_hex_to_base64/encode_base64.cpp

```diff
@@ -29,51 +29,43 @@ std::ostream & operator<<(std::ostream & output,
     }
 
     // First base64 digit is composed of high 6 bits of first octet
-    output <<
-            encode_base64::convertToBase64(
-                    *manipulator.d_input[0]
-                    >> (
-                            encode_base64::s_numBitsInOctet
-                            - encode_base64::s_numBitsInBase64Digit
-                       )
-            );
+    output << encode_base64::convertToBase64(
+        *manipulator.d_input[0] >> (encode_base64::s_numBitsInOctet
+                                    - encode_base64::s_numBitsInBase64Digit));
 
     // Second base64 digit is composed of:
     //  - low 2 bits of first octet
     //  - high 4 bits of second octet
     static const size_t s_numRemainingBitsFromFirstOctet =
-            encode_base64::s_numBitsInOctet
-            - encode_base64::s_numBitsInBase64Digit;
+        encode_base64::s_numBitsInOctet - encode_base64::s_numBitsInBase64Digit;
     static const size_t s_numBitsFromSecondOctet =
-            encode_base64::s_numBitsInBase64Digit
-            - s_numRemainingBitsFromFirstOctet;
+        encode_base64::s_numBitsInBase64Digit
+        - s_numRemainingBitsFromFirstOctet;
     uint8_t inputBitsForSecondBase64Digit =
-            (
-                    *manipulator.d_input[0]
-                    & ((1u << s_numRemainingBitsFromFirstOctet) - 1)
-            ) << s_numBitsFromSecondOctet;
+        (*manipulator.d_input[0]
+         & ((1u << s_numRemainingBitsFromFirstOctet) - 1))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Uppercase the literal suffix. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3ALiteralSuffix)